### PR TITLE
[windows] Fix mapping/pipelines for time_created

### PR DESCRIPTION
**Pull request by nicpenning**
- Bug
- Enhancement

## What does this PR do?

Align with other Windows integrations and the winlog.time_created improper mapping and adjusting the error handling for said pipelines.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

Link related issues below. Insert the issue link or reference after the word "Closes" if merging this should automatically close it.

- Relates #5350

### CODE REVIEW

**Comment by nicpenning** [comment]
I got a little wild trying to find a replace all areas where the winlog.time_created existed and accidently removed the target field of event.created that this should go into which is interesting because most places it does not do that. There seems to be more than what meets the eye with this time_created debacle. @efd6 👋

**Comment by nicpenning** [comment]
Also, there is this issue:
image

Any idea where in the code this can get addressed for the Forwarded logs index template?

File: packages/windows/data_stream/forwarded/elasticsearch/ingest_pipeline/powershell.yml [hunk 1]
```diff
@@ -31,13 +31,21 @@ processors:
       ignore_empty_value: true
       ignore_failure: true
       if: ctx?.winlog?.level != ""
-  - date:
+- date:
```
Comment (efd6) [inline packages/windows/data_stream/forwarded/elasticsearch/ingest_pipeline/powershell.yml:34]:
```suggestion
  - date:
```

Comment (nicpenning) [reply packages/windows/data_stream/forwarded/elasticsearch/ingest_pipeline/powershell.yml]: Good eye

**Review by efd6 — COMMENTED** [review]
I can't see field definitions for `winlog.time_created` in powershell, powershell_operational or sysmon_operational. It should be there too, no?

**Comment by nicpenning** [comment]
I can check. In some cases the pipelines remove that field entirely.

**Comment by nicpenning** [comment]
Confirmed, each of those remove this field so it should not exist.

**Review by efd6 — APPROVED** [review]

**Pull request merged**